--- a/CMakeLists.txt	2025-09-27 08:40:06.000000000 +0200
+++ b/CMakeLists.txt	2025-10-24 20:59:16.490198241 +0200
@@ -305,16 +305,10 @@
 	#SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-unused-function")			#
 #	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wswitch-enum")					#to warn about omitted enums despite default.
 
-	#might as well do this, public builds use the regular Makefile.
-	IF(CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64le" OR CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64")
-		SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mcpu=native")
-	ELSE()
-		SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -march=native")
-	ENDIF()
 	IF(CMAKE_BUILD_TYPE MATCHES "Debug")
 		set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall ${FTE_WERROR_} -Wno-pointer-sign -Wno-unknown-pragmas -Wno-format-zero-length -Wno-strict-aliasing -Wno-error=cpp")
 	ELSE()
-		set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O3 ${FTE_WERROR_}")
+		set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${FTE_WERROR_}")
 	ENDIF()
 	IF (NOT FTE_USE_SDL)
 		set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wl,--warn-common")
